lib/beaker-hostgenerator/cli.rb

```diff
@@ -121,7 +121,7 @@ def initialize(argv = ARGV.dup)
         raise BeakerHostGenerator::Exceptions::SafeEarlyExit
       else
         # Tokenizing the config definition for great justice
-        @tokens = argv[0].split('-')
+        @tokens = __tokenize_input(argv[0])
```

Comment:
Not a big fan of this naming convention for internal/private methods in ruby. I would either declare this `private` as `tokenize_input` or extract out to a class (seems like extraction is a a bit much to do for this small method, though).